test/es-module/test-esm-import-attributes-identity.mjs

```diff
@@ -0,0 +1,52 @@
+import '../common/index.mjs';
+import assert from 'node:assert';
+import Module from 'node:module';
```

Comment:
I wasn't aware of a convention to import `Module` instead of `module` here. What makes `Module` different from other core libs here? Seems it should use the same casing unless there is a good reason.

Reply:
It's just my preference lasted from CJS that to prevent name collision with CJS `module`.

Reply:
In `node:module`, `module.exports = Module`, so the default export is also the `Module` constructor.